### PR TITLE
Outline the bugfix backport process

### PR DESCRIPTION
**Pull request by mdellweg**
[noissue]

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -11,3 +11,12 @@ A Pulp plugin to support hosting arbitrary files.
 
 For more information, please see the `documentation <https://docs.pulpproject.org/pulp_file/>`_
 or the `Pulp project page <https://pulpproject.org>`_.
+
+Development moved to Pulpcore
+-----------------------------
+
+Since this plugin has been moved to pulpcore, issues can no longer be reported here. Please go to
+`Pulpcore - Issues<https://github.com/pulp/pulpcore/issues>`_ instead.
+Any issues that asrise should also be solved there first and if needed backported to here. We do
```
Comment (ipanova) [inline README.rst:20]: s/asrise/arise

Comment (ipanova) [reply README.rst:20]: s/backported to here/backported here

Comment (ipanova) [reply README.rst:20]: I would drop last 3 sentences

File: README.rst [hunk 2]
```diff
@@ -11,3 +11,12 @@ A Pulp plugin to support hosting arbitrary files.
...
 or the `Pulp project page <https://pulpproject.org>`_.
+
+Development moved to Pulpcore
+-----------------------------
+
+Since this plugin has been moved to pulpcore, issues can no longer be reported here. Please go to
+`Pulpcore - Issues<https://github.com/pulp/pulpcore/issues>`_ instead.
+Any issues that arise should also be solved there first and if needed backported to here. We do
+neither expect this to happen soon, nor often. So we will work out the details when / if that
+happens. Please excuse if this takes more time than estimated.
```
Comment (ggainey) [inline README.rst:22]:
```suggestion
Any issues that arise should also be solved there first and if needed backported to here. We expect this to happen only infrequently, if ever. We will therefore  work out the details when / if it
happens. Please excuse us if this takes more time than desired.
```

**Review by ggainey — CHANGES_REQUESTED** [review]

File: README.rst (hunk 2 above)
Comment (ggainey) [reply README.rst:22]: Dropping it sounds good, for exactly @ipanova 's reasoning

**Review by ipanova — APPROVED** [review]

**Review by ggainey — APPROVED** [review]

**Pull request merged**